rastervision/command/chip_command_config.py

```diff
@@ -75,17 +77,29 @@ def validate(self):
         if self.task is None:
             raise rv.ConfigError('Task not set for ChipCommandConfig. Use '
                                  'with_task or with_experiment')
+        check_task_type(self.task)
         if self.backend is None:
             raise rv.ConfigError('Backend not set for ChipCommandConfig. Use '
                                  'with_backend or with_experiment')
+        check_backend_type(self.backend)
         if self.train_scenes == []:
             raise rv.ConfigError(
                 'Train scenes not set for ChipCommandConfig. Use '
                 'with_train_scenes or with_experiment')
+        if len(self.train_scenes) > 0:
+            if not isinstance(self.train_scenes[0], SceneConfig):
+                raise rv.ConfigError(
```

Comment:
It would be preferable loop through the whole list and check each item, rather than only check the first one. (Same comment applies to some other spots with lists.)

Reply:
yeah, that makes sense. I made these changes where necessary